GUI/src/main/kotlin/ui/screens/SelectVideoScreen.kt

```diff
@@ -94,3 +104,38 @@ fun RowScope.AdvancedSettingsButton(state: MutableState<AppState>) {
         )
     }
 }
+
+@Composable
+fun helpMenu(modifier: Modifier = Modifier) {
+    var expanded by remember { mutableStateOf(false) }
+    Button(
+        modifier = modifier,
+        onClick = { expanded = !expanded },
+    ) {
+        Text("?", fontSize = MaterialTheme.typography.body2.fontSize)
+    }
+
+    DropdownMenu(
+        modifier = modifier.padding(8.dp),
+        expanded = expanded,
+        onDismissRequest = { expanded = false },
+    ) {
+        hyperlinkDropdownMenuItem("Project Page", "https://github.com/amosproj/amos2023ws03-gui-frame-diff")
+        hyperlinkDropdownMenuItem("Help", "https://github.com/amosproj/amos2023ws03-gui-frame-diff/blob/main/README.md")
```

Comment:
i would link to the wiki as it includes some more documentation aswell

Reply:
outdated